apps/me/Dockerfile

```
@@ -1,43 +1,46 @@
 ENV PNPM_HOME="/pnpm"
 ENV PATH="$PNPM_HOME:$PATH"
+ENV CI=true
+ENV SKIP_ENV_VALIDATION=true
 
 RUN apk add --no-cache libc6-compat
 WORKDIR /app
 
-RUN corepack enable && corepack prepare pnpm@8.15.1 --activate
-RUN pnpm add -g turbo@1.12.3
+RUN npm install -g turbo@1.13.4
+RUN corepack enable
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)